<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>thyroid.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="thysysName">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 127);
font: 87 14pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>THY-SYS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(85, 170, 255);
border-radius: 10px;

</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(85, 170, 255);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_password">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(85, 170, 255);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmPasswordLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Signup">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255);
font: 75 9pt &quot;Arial&quot;;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(156, 156, 156);</string>
   </property>
   <property name="text">
    <string>Already have an account?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginAcc">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string>Log In Now</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
